assets/js/components/DataBlockGroup.js

```diff
@@ -96,11 +98,30 @@ export default function DataBlockGroup( { className, children } ) {
 		} );
 	};
 
+	const resetFontSizes = async ( blocks ) => {
```

Comment:
@ankitrox @benbowler, it's not clear to me why this needs to be `async`, could you explain why?